api/controllers/search.js

```diff
@@ -73,10 +73,11 @@ router.get("/getAllForType", async function getAllForType(req, res) {
 // strip off final character (assumed to be "s")
 const singularLowerCase = name =>
   (name.slice(-1) === "s" ? name.slice(0, -1) : name).toLowerCase();
-// like it says on the tin
-const filterFromReq = req => {
-  const cat = singularLowerCase(req.query.selectedCategory || "Alls");
-  return cat === "all" ? "" : `AND type = '${cat}'`;
+
+// just get the type, if specified
+const typeFromReq = req => {
+  let cat = singularLowerCase(req.query.selectedCategory || "Alls");
```

Comment:
looks like the fallback should be `all` here

Reply:
The first thing we do is strip off the last letter of Cases, Methods, etc, thus Alls